.cicd/pipeline.yml

```diff
@@ -1,5 +1,4 @@
 steps:
-  - wait
```

Comment:
We should leave a wait step at the beginning of the pipeline steps to protect against errors. Without it, the steps after "Pipeline Upload" will begin running before the pipeline upload step is done. This might not matter for contracts or eos-vm, but it will introduce an opportunity for error in EOSIO when we are dynamically constructing pipeline steps for container-level parallelization.

Reply:
Done.